apps/hash-frontend/src/pages/shared/top-context-bar.tsx

```diff
@@ -84,39 +94,35 @@ const pageRestoredMessage = (
       })}
     />
     <Typography variant="smallTextParagraphs" sx={{ fontWeight: 500 }}>
-      Page restored
+      {children}
     </Typography>
   </Box>
 );
 
 type TopContextBarProps = {
   crumbs: BreadcrumbsProps["crumbs"];
+  item?: Entity | EntityTypeWithMetadata;
   defaultCrumbIcon?: ReactNode;
-  isBlockPage?: boolean;
   scrollToTop: () => void;
-  displayPageRestoredMessage?: boolean;
   sx?: SxProps<Theme>;
 };
 
 export const TOP_CONTEXT_BAR_HEIGHT = 50;
 
 export const TopContextBar = ({
```

Comment:
Not suggesting we do it in this PR, but this bar is becoming sufficiently complex that I think it's possibly going to end up better to have it more as a container which different routes can trigger adding content to somehow, rather than trying to cover every possible thing we want to show in it inside this component